Mixpanel/MPNotificationViewController.m

```diff
@@ -440,6 +449,10 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder
     if (self = [super initWithCoder:aDecoder]) {
         _maskLayer = [GradientMaskLayer layer];
         [self.layer setMask:_maskLayer];
+        [_maskLayer setColors:@[(id)[UIColor blackColor].CGColor,(id) [UIColor blackColor].CGColor,(id) [UIColor clearColor].CGColor,(id) [UIColor clearColor].CGColor]];
```

Comment:
why did you have to cast to id? is that safe to do with CGColor? and if you need an object why not just use the UIColor that you're originally getting the CGColor from?